example_notebooks/moving_pictures/moving_pictures.ipynb

```diff
@@ -0,0 +1,413 @@
+    "**Example:** selecting the log-ratio of the top 10% to bottom 10% of features for a given ranking.\n",
+    "\n",
+    "**In practice:** this is really useful for quickly looking at the top- and bottom-ranked features for a given ranking -- for example, if you want to see \"how well these rankings separate samples for a given metadata field.\"\n",
+    "\n",
+    "## 9.2. Filtering\n",
+    "\n",
+    "Using **filtering**, you can select features based on focused searches through their IDs or metadata. The searches can be textual (e.g. selecting features where the `taxonomy` field contains the text `Ileibacterium`) or numeric (e.g. selecting features where some feature metadata value is less than 5).\n",
+    "\n",
+    "**Example:** selecting the log-ratio of all features with taxonomy annotations containing the text `Staphylococcus` over all features with taxonomy annotations containing the text `Propionibacterium`.\n",
+    "\n",
+    "**In practice:** this is useful for doing more targeted investigations than autoselection, since you have much more control over what features you include. For example, if a certain group of taxa seems especially highly- or lowly-ranked for a given ranking of interest, you can try filtering to that group here to see how all of the observed members of that taxonomic group are distributed throughout the rankings.\n",
+    "\n",
+    "## 9.3. Clicking\n",
+    "\n",
+    "It's pretty basic compared to the other selection methods, but by **clicking on the rank plot twice** you can select simple log-ratios of single features. The first click sets the numerator feature for a log-ratio, and the second click sets the denominator feature for the log-ratio.\n",
+    "\n",
+    "**Example:** if you have a small number of features, and you just want to look at a log-ratio of two features quickly. (The [color data tutorial](https://nbviewer.jupyter.org/github/biocore/qurro/blob/master/example_notebooks/color_compositions/color_example.ipynb) is a good example of this.) \n",
+    "\n",
+    "**In practice:** this method isn't that useful compared to the other two, especially for complicated datasets. If you're having a hard time clicking on features, increasing the bar width via the slider below the rank plot can help with this.\n",
+    "\n",
+    "## 9.4. Sidenote: computing log-ratios involving > 2 features\n",
+    "In all of the selection methods above except for the \"clicking\" method, an arbitrary number of features can be present in the numerator or denominator of the log-ratio.\n",
+    "\n",
+    "In this case, the log-ratio is computed for a given sample by summing the abundances of the numerator features, summing the abundances of the denominator features, and then taking the log-ratio of these sums. Written out as a formula, this is `log(top sum) - log(bottom sum)` (or, [equivalently](https://en.wikipedia.org/wiki/List_of_logarithmic_identities#Using_simpler_operations), `log(top sum / bottom sum)`).\n",
```

Comment:
Would clarify natural log here.

Reply:
Changed to `ln()` instead of `log()` in the formulae.